framework/src/actions/CommonOutputAction.C

```diff
@@ -99,10 +99,12 @@ CommonOutputAction::validParams()
 
   params.addParam<std::vector<VariableName>>(
       "hide",
+      std::vector<VariableName>{},
```

Comment:
it would be cool if we could support a `std::initializer_list<T>` here for parameter type `std::vector<T>`...

Reply:
`std::initializer_list<T>` could be a good idea. I fixed almost all cases in framework and modules, but the apps may be an issue here, especially I don't have access to many of them